SixTest/stf_output/stf/new_converter.f

```diff
@@ -0,0 +1,80 @@
+      program new_converter
```

Comment:
In addition to adding the descriptive comment mentioned in the `old_converter.f`:

For the new converter, having an optional command line argument specifying which particle(s) to extract - right now it's only ever pair 1 if I understand your code correctly - would be very useful. For the old converter, maybe just use the filename as an argument?

I would also propose to name your output files as something like "singletrackfile.dat.PAIRNUMBER" (same goes for those extracted from the old format).